docker-entrypoint.sh

```
@@ -0,0 +1,30 @@
+echo "Starting Next.js server on port ${PORT:-3000}..."
+bun server.js &
+NEXT_PID=$!
+
+# Handle shutdown signals
+cleanup() {
+  echo "Shutting down servers..."
+  kill $ELYSIA_PID $NEXT_PID 2>/dev/null || true
+  exit 0
+}
+
+trap cleanup SIGTERM SIGINT
```

Shellcheck (0.11.0)

[warning] In POSIX sh, prefixing signal names with 'SIG' is undefined.

(SC3048)